lib/data-types.d.ts

```diff
@@ -396,105 +512,128 @@ export class VIRTUAL<T> extends ABSTRACT {
   type: T;
 }
 
 /**
  * A geometry datatype represents two dimensional spacial objects.
  * @property GEOMETRY
  */
-export class GEOMETRY extends ABSTRACT {
-  constructor(type, srid);
-  constructor(options: GEOMETRY_Options);
+export const GEOMETRY: GeometryDataTypeStatic;
 
-  options: GEOMETRY_Options;
-  type: any;
-  srid: any;
+export interface GeometryDataTypeStatic extends AbstractDataTypeStatic {
+  new (type: string, srid?: number): GeometryDataType;
+  new (options: GeometryDataTypeOptions): GeometryDataType;
+  (type: string, srid?: number): GeometryDataType;
+  (options: GeometryDataTypeOptions): GeometryDataType;
+}
 
+export interface GeometryDataType extends AbstractDataType {
+  options: GeometryDataTypeOptions;
+  type: string;
+  srid?: number;
   escape: boolean;
-  $stringify(value: any): string;
 }
-export interface GEOMETRY_Options {
-  type: any;
-  srid: any;
+
+export interface GeometryDataTypeOptions {
+  type: string;
+  srid?: number;
 }
 
 /**
  * A geography datatype represents two dimensional spacial objects in an elliptic coord system.
  * @property GEOGRAPHY
  */
-export class GEOGRAPHY extends ABSTRACT {
-  construcor(type, srid);
-  construcor(options: GEOGRAPHY_Options);
-
-  options: GEOGRAPHY_Options
+export const GEOGRAPHY: GeographyDataTypeStatic;
 
-  type: any;
-  srid: any;
+export interface GeographyDataTypeStatic extends AbstractDataTypeStatic {
+  new (type: string, srid?: number): GeographyDataType;
+  new (options: GeographyDataTypeOptions): GeographyDataType;
+  (type: string, srid?: number): GeographyDataType;
+  (options: GeographyDataTypeOptions): GeographyDataType;
+}
 
+export interface GeographyDataType extends AbstractDataType {
+  options: GeographyDataTypeOptions;
+  type: string;
+  srid?: number;
   escape: boolean;
-  $stringify(value: any): string;
 }
-export interface GEOGRAPHY_Options {
-  type: any;
-  srid: any;
+
+export interface GeographyDataTypeOptions {
+  type: string;
+  srid?: number;
 }
 
 export const NONE: typeof VIRTUAL;
 // export ['DOUBLE PRECISION']: typeof DOUBLE;
 
 export interface DataTypes {
-  ABSTRACT: typeof ABSTRACT;
-  STRING: typeof STRING;
-  CHAR: typeof CHAR;
-  TEXT: typeof TEXT;
-  NUMBER: typeof NUMBER;
-  INTEGER: typeof INTEGER;
-  BIGINT: typeof BIGINT;
-  FLOAT: typeof FLOAT;
-  REAL: typeof REAL;
-  DOUBLE: typeof DOUBLE;
-  DECIMAL: typeof DECIMAL;
-  BOOLEAN: typeof BOOLEAN;
-  TIME: typeof TIME;
-  DATE: typeof DATE;
-  DATEONLY: typeof DATEONLY;
-  HSTORE: typeof HSTORE;
-  JSON: typeof JSON;
-  JSONB: typeof JSONB;
-  NOW: typeof NOW;
-  BLOB: typeof BLOB;
-  RANGE: typeof RANGE;
-  UUID: typeof UUID;
-  UUIDV1: typeof UUIDV1;
-  UUIDV4: typeof UUIDV4;
-  VIRTUAL: typeof VIRTUAL;
-  ENUM: typeof ENUM;
-  ARRAY: typeof ARRAY;
-  GEOMETRY: typeof GEOMETRY;
-  GEOGRAPHY: typeof GEOGRAPHY;
-  'DOUBLE PRECISION': typeof DOUBLE;
+    ABSTRACT: typeof ABSTRACT;
```

Comment:
I think my editor did it because it was using tabs while the rest of the file(s) use spaces